src/cleanlab_codex/validator.py

```diff
@@ -81,13 +83,14 @@ def __init__(
 
         self._bad_response_thresholds = BadResponseThresholds.model_validate(bad_response_thresholds or {})
 
-        _threshold_keys = self._bad_response_thresholds.model_dump().keys()
-
-        # Check if there are any thresholds without corresponding evals (this is an error)
-        _extra_thresholds = set(_threshold_keys) - set(_evals)
-        if _extra_thresholds:
-            error_msg = f"Found thresholds for non-existent evaluation metrics: {_extra_thresholds}"
-            raise ValueError(error_msg)
+        # Only check thresholds that were explicitly provided by the user
+        if bad_response_thresholds is not None:
+            _threshold_keys = bad_response_thresholds.keys()
+            # Check if there are any user-provided thresholds without corresponding evals
+            _extra_thresholds = set(_threshold_keys) - set(_evals)
+            if _extra_thresholds:
+                error_msg = f"Found thresholds for non-existent evaluation metrics: {_extra_thresholds}"
```

Comment:
SUPER nit, i have no strong opinion for the way you had it originally but this tells you more where to look to fix the issue
```suggestion
                error_msg = f"Found thresholds for evaluation metrics not specified in evals parameter: {_extra_thresholds}"
```

Reply:
Thanks for the feedback! I agree about making the error message more helpful for users configuring TrustworthyRAG. I've updated it to be more specific about where to look.